### PR TITLE
only mark trash phrases as complete if they have no votes

### PR DESCRIPTION
**Pull request by wilmol**
- could change to 1-2 votes
- think its unfair if someone votes and someone else trashes after
- would also prevent deadlocked phrase from being resolved early

### CODE REVIEW

File: concord-server/server/src/main/java/com/rabidgremlin/concord/dao/VotesDao.java [hunk 1]
```diff
@@ -86,4 +86,7 @@
   @RegisterBeanMapper(GroupedPhraseVoteWithMostRecentVoteTime.class)
   List<GroupedPhraseVoteWithMostRecentVoteTime> getLabelsForUncompletedPhrasesWithMostRecentVoteTime();
 
+  @SqlQuery("SELECT COUNT(*) FROM votes WHERE phraseId = :phraseId")
+  int getVoteCountForPhrase(@Bind("phraseId") String phraseId);
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/dao/VotesDao.java]: We'd want to ignore BULK UPLOAD votes here wouldn't we?

Comment (wilmol) [reply concord-server/server/src/main/java/com/rabidgremlin/concord/dao/VotesDao.java]: yes

File: concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java [hunk 2]
```diff
@@ -215,15 +215,14 @@ public Response voteForPhrase(@ApiParam(hidden = true) @Auth Caller caller, @Pat
...
-
     votesDao.upsert(phraseId, label, userId);
 
     // are we marking trashed phrases as completed?
     if (completeOnTrash && StringUtils.equals(LABEL_TRASH, label))
     {
       // only if the phrase hasn't been voted on yet
-      if (voteCountForPhrase <= 0)
+      int voteCountForPhrase = votesDao.getVoteCountForPhrase(phraseId);
+      if (voteCountForPhrase <= 1)
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java]: Is the upsert guaranteed to complete before we get the vote counts?

Comment (wilmol) [reply concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java]: it blocks

**Comment by wilmol** [comment]
@rabidgremlin the CI didnt run ?

File: concord-ui/src/components/systemstatstable.js [hunk 3]
```diff
@@ -139,7 +139,7 @@ export class SystemStatsTable extends Component {
       <div>
         <SystemStatsTable />
         <Typography style={{ fontSize: '30px' }} use='subtitle1' tag='h2'>
-          {deadLockedPhrases.length} Deadlocked Phrases
+          {deadLockedPhrases.length.toLocaleString()} Deadlocked Phrases
```
Comment (LaurenceTews) [inline concord-ui/src/components/systemstatstable.js]: What's this for?

Comment (wilmol) [reply concord-ui/src/components/systemstatstable.js]: will format the string, e.g. 1000 -> 1,000

**Pull request merged**